iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/HelmParser.java

```diff
@@ -95,7 +95,7 @@ private FileTree evaluateAndParseHelmFile(String source, HelmInputFileContext in
   }
 
   static boolean isInvalidHelmInputFile(HelmInputFileContext helmFileCtx) {
-    return isValuesFile(helmFileCtx) || isChartFile(helmFileCtx) || isTplFile(helmFileCtx);
+    return isValuesFile(helmFileCtx) || isTplFile(helmFileCtx);
```

Comment:
I guess that thanks to this change we will be able now to raise issues in Chart.yaml 🤔 
Please verify that, and if yes, then please add a test case for some rule, e.g. TODO check.

Reply:
Yes, added a TODO comment in ITs